src/main/resources/SnakemakeBundle.properties

```diff
@@ -22,3 +22,7 @@ INSP.NAME.shadow.settings=Shadow must either be 'shallow', 'full', 'minimal', or
 
 # SnakemakeMultipleShadowSettingsInspection
 INSP.NAME.shadow.multiple.settings=Only one argument after shadow section is permitted.
+
+# SmkSectionRedeclarationInspectoin
+INSP.NAME.section.redeclaration=Section declaration above overshadows this declaration
```

Comment:
According to the Oxford dictionary "overshadow - to make smb/smth seem less important, or successful", is it a right verb for this case? Maybe just "overrides"?